packages/client/src/mud/supportedChains.ts

```diff
@@ -7,22 +7,34 @@
  *   basefee set to zero to avoid transaction fees.
  *
  */
-import { MUDChain, mudFoundry, garnet, redstone } from "@latticexyz/common/chains"
+import {
+  MUDChain,
+  mudFoundry,
+  garnet,
+  redstone,
+} from "@latticexyz/common/chains";
+import { holesky, mainnet } from "viem/chains";
 
 type ExtendedChain = MUDChain & {
-  indexerUrl?: string | undefined
-}
+  indexerUrl?: string | undefined;
+};
 
-const extendedGarnet: ExtendedChain = garnet as ExtendedChain;
-extendedGarnet.faucetUrl = "https://17001-faucet.quarry.linfra.xyz/trpc/drip"
-extendedGarnet.indexerUrl = "https://indexer.mud.garnetchain.com/ "
+const extendedGarnet = {
+  ...garnet,
+  faucetUrl: "https://17001-faucet.quarry.linfra.xyz/trpc/drip",
+  indexerUrl: "https://indexer.mud.garnetchain.com/",
+} as const satisfies ExtendedChain;
```

Comment:
the previous approach you had here was mutating the original object, which isn't great

this new approach should be more runtime safe and strongly typed